docs/proposals/28-autonomous-shoot-clusters.md

```diff
@@ -0,0 +1,497 @@
+Depending on the `etcd` management, it may be necessary to have individual `OperatingSystemConfig`s for each control plane node.
+However, this is not yet fully clear and needs further investigation (see [`etcd` Management via `etcd-druid`](#etcd-management-via-etcd-druid)).
+Apart from this, `etcd` also needs to be re-configured to have the new node join the `etcd` cluster.
+
+#### `gardenadm connect`
+
+This command deploys a `gardenlet` into the autonomous shoot cluster and connects it to an existing garden cluster.
+As part of this operation, the corresponding `Shoot` resource will be created in the garden cluster.
+Afterwards, the `gardenlet` will reconcile the autonomous shoot cluster similarly to regular shoot clusters.
+
+> [!NOTE]
+> When there is no existing Gardener system yet (for example, because the created autonomous shoot cluster is the initial cluster), then [`gardener-operator`](../concepts/operator.md) must be deployed first, and the [`Garden`](../../example/operator/20-garden.yaml) must be created, before the `gardenlet` can be deployed.
+>
+> Also, it is not yet clear whether the existing `gardenlet` binary will be re-used, or whether we better create a dedicated, new binary with slimmed down functionality catered for only managing autonomous shoot clusters.
+
+Depending on the scenario, an autonomous shoot cluster may be air-gapped for periods of time.
+Therefore, please note that the cluster should function also when air-gapped/when it has limited or no network connectivity.
+Updates can not be performed during that time, but ordinary operations with the cluster itself (e.g., creating pods or services) should still be possible.
+It is important that the [version skew](../deployment/version_skew_policy.md) is always respected.
+
+#### `gardenadm version`
+
+This command prints the version of `gardenadm`.
+
+### `Shoot` API
+
+We need some mechanism to distinguish regular shoot clusters from autonomous shoot clusters.
+As the most prominent difference is that there are dedicated machines for the control plane components, we propose to notice the difference based on the existence of a worker pool for the control plane:
+
+```yaml
+spec:
+  provider:
+    workers:
+    - name: control-plane
+      controlPlane: {}
+```
+
+An alternative is to augment `.spec.controlPlane` with this information, however, we think adding a dedicated worker pool is the better and more natural fit (we need it anyways to enable re-use of most of the existing code).
+
+### Scenarios
+
+We distinguish between two different scenarios (for now):
+
+- [High Touch](#high-touch), meaning that there is no programmable infrastructure available.
+  We consider this the "bare metal" or "edge" use-case, where at first machines must be (often manually) prepared by human operators.
+  In this case, network setup or machine management is out of scope.
```

Comment:
Just to be sure: in this line, the term "network setup" refers to setting up the host/node network (i.e., VPCs and stuff) and not the pod network correct?